tests/test-integration-mock.py

```diff
@@ -544,14 +544,13 @@ def test_07_tbtadm_approve_sl2(self):
     # Test multi - controller device tree
     def test_08_multi_controller_topology(self):
         # connect all device
-        device1 = TbDevice("Device1")
-        device2 = TbDevice("Device2", children = [device1])
+        device1 = TbDevice("0-2")
```

Comment:
This should be using the correct port numbers. So they are 0-1 (for the first port) and 0-3 (for the second port).

Reply:
@westeri here device1 is children of device2

Reply:
In that case you would have 0-101 or 0-103 or so depending on which port the device is connected.

Reply:
@westeri
I don't think this is guaranteed by HW/FW. The best way to handle this is to accept both lanes for the port (1-2 are mapped to port A, 3-4 - to port B).

@finikorg
If device1 is child of device2 it can't have 0-2 as route string.

Reply:
Actually, it means the code is "connecting" both devices to the same port (1 and 2 are both lanes of port A, as explained above), something that is physically impossible of course. It doesn't matter much from user-space perspective, we trust sysfs info from the driver, but this is not a real world case.

Reply:
It actually is guaranteed by the firmware to always enumerate device through the primary port (1 or 3). Only exception are some legacy devices. I would stick with that here.